Library/Homebrew/dev-cmd/edit.rb

```diff
@@ -40,16 +40,24 @@ def edit
       EOS
     end
 
-    paths = args.named.to_paths.select do |path|
-      next path if path.exist?
+    paths = if args.named.empty?
+      # Sublime requires opting into the project editing path,
+      # as opposed to VS Code which will infer from the .vscode path
+      if which_editor == "subl"
```

Comment:
Do we have many maintainers/contributors using Sublime? This feels very specific to a single editor and perhaps single maintainer?

Reply:
I'm not the only maintainer using Sublime. You can see references to other maintainers using it both in Slack and on the issue tracker. I may be the only one using Sublime on Apple silicon for now, which is why I'm the first to run into this issue, but I won't be forever as people upgrade their hardware.

Reply:
In general: given this is a developer-specific command, and the diff is quite small, I don't think it's a concern to carry this code. There is no real risk, and the command exists to support Homebrew developers like myself. As I mentioned before, we carried more complex code than this for close to a decade without it ever having caused a problem.